db/index.js

```diff
@@ -1,12 +1,66 @@
 const mysql = require('mysql');
 const config = require('./config.js');
-
+const faker = require('faker');
+// const data = require('./data-functions.js')
 const connection = mysql.createConnection(config);
 
+const createFakeUser = () => ({
+  username: faker.internet.userName(),
+  // avatar: faker.image.imageUrl()
+});
+
+// functions to query database
+const seedUsers = async (func) => {
+  for (let i = 0; i < 100; i++) {
+    let queryString = 'INSERT INTO users (username) VALUES (?)'
+    let user = createFakeUser();
+    connection.query(queryString, user.username);
+  }
+  await createReviews();
+}
+
+const createReviews = async () => {
+  let reviews = [];
```

Comment:
Should these be set to i < 99? Curious why these are set to i < 100 while it's set to i < 99 in your data.js file